pom.xml

```diff
@@ -8,9 +8,15 @@
     <artifactId>parent</artifactId>
     <version>1.0-SNAPSHOT</version>
     <properties>
-        <jacoco-maven-plugin.version>0.7.5.201505241946</jacoco-maven-plugin.version>
+        <jacoco-maven-plugin.version>0.7.9</jacoco-maven-plugin.version>
         <maven-resources-plugin.version>2.7</maven-resources-plugin.version>
         <maven-surefire-plugin.version>2.17</maven-surefire-plugin.version>
+
+        <!-- The destination file for the code coverage report has to be set to the same value
+         in the parent pom and in each module pom.
+         Then Sonar will use the report containing IT coverage when running Sonar analysis in each source modules.
+         -->
+        <sonar.jacoco.itReportPath>${project.basedir}/../integration-tests/target/jacoco-it.exec</sonar.jacoco.itReportPath>
```

Comment:
This is the trick: since sonar can't use the it-coverage result file (jacoco-it.exec) while scanning the integration-tests module (because of the lack of sources of the tested classes), we're making the it-coverage result file available during all modules scanning. That way sonar will consider it-tests results while scanning the client and backend modules and thus the global coverage will be correct.